ui/packages/atlasmap-core/src/models/config.model.ts

```diff
@@ -268,6 +268,14 @@ export class ConfigModel {
     return this.getDocs(isSource).find((d) => d.id === documentId)!;
   }
 
+  getDocForName(
```

Comment:
IIRC we allow duplicated document name, how do we distinguish them?

Reply:
removed - this function is not needed